package.json

```diff
@@ -33,5 +34,9 @@
   "dependencies": {
     "node-static": "^0.7.11",
     "portfinder": "^1.0.32"
+  },
+  "volta": {
+    "node": "20.5.0",
+    "npm": "9.8.0"
```

Comment:
Up to you if you want to remove this. If you put this in your package.json and someone is using Volta, then it will automatically use these node/npm versions when you are in this project's folder. This allows enforcing a matching Node version to what is built in to NW.js. It is easy to run into problems if your Node version does not match NW.js.